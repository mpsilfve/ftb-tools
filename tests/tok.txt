Vuosi 1919. Se 2. luokka
