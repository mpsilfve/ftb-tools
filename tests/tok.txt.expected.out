
"<Vuosi>"
	"Vuo" N Prop Pl Nom PxSg2 Abc #1
	"Vuo" N Prop Sg Gen PxSg2 Abc #1
	"Vuo" N Prop Sg Nom PxSg2 Abc #1
	"vuo" N Pl Nom PxSg2 Abc #1
	"vuo" N Sg Gen PxSg2 Abc #1
	"vuo" N Sg Nom PxSg2 Abc #1
	"vuosi" N Sg Nom Abc #1
	"vuotaa" V Act Ind Past Sg3 Abc #1
"<1919>"
	"1919" Num Card noabc
"<.>"
	"." Pun noabc #1
"<Se>"
	"se" Pron Dem Sg Nom Abc #1
"<2.>"
	"2." Num Ord noabc
"<luokka>"
	"luokka" N Sg Nom abc #1
